--- BUILD	2023-03-15 12:12:00.000000000 +0000
+++ BUILD_NEW	2023-03-15 12:11:44.000000000 +0000
@@ -52,9 +52,26 @@
         "darwin": ":cc-compiler-darwin",
         "armeabi-v7a|compiler": ":cc-compiler-armeabi-v7a",
         "armeabi-v7a": ":cc-compiler-armeabi-v7a",
+        "darwin_arm64|clang": ":cc-compiler-darwin",
+        "darwin_arm64": ":cc-compiler-darwin",
     },
 )

+cc_toolchain(
+    name = "cc-compiler-darwin_arm64",
+    toolchain_identifier = "local",
+    toolchain_config = ":local",
+    all_files = ":compiler_deps",
+    ar_files = ":compiler_deps",
+    as_files = ":compiler_deps",
+    compiler_files = ":compiler_deps",
+    dwp_files = ":empty",
+    linker_files = ":compiler_deps",
+    objcopy_files = ":empty",
+    strip_files = ":empty",
+    supports_param_files = 1,
+    module_map = ":module.modulemap",
+)
 cc_toolchain(
     name = "cc-compiler-darwin",
     toolchain_identifier = "local",
